DRAFT.md

```diff
@@ -1380,9 +1388,9 @@ constexpr polymorphic& operator=(polymorphic&& other) noexcept(
   Otherwise if `alloc != other.alloc`; if `other` is not valueless, a new owned
   object is constructed in `this` using
   `allocator_traits<allocator_type>::construct` with the owned object from
-  `other` as the argument as an rvalue, with memory allocated using either the
-  allocator in `this` or the allocator in `other` if the allocator needs
-  updating. The previous owned object in this, if any, is destroyed using
+  `other` as the argument as an rvalue, using either the allocator in `this` or
+  the allocator in `other` if the allocator needs updating. The previous owned
+  object in this, if any, is destroyed using
   `allocator_traits<allocator_type>::destroy` and then deallocated using
   `allocator_traits<allocator_type>::deallocate`.
```

Comment:
As noted above, this should be  `allocator_traits<allocator_type>::rebind_traits::deallocate` but maybe just say "the storage is deallocated" instead.